### PR TITLE
Add "Release Versions Only" post.

### PR DESCRIPTION
**Pull request by floere**
I'd be happy for any stylistic improvements, typo pointers, and general feedback :)

### CODE REVIEW

**Comment by kylef** [comment]
Perhaps it would be nice to include a before/after screenshot? Especially considering it's a short post.

**Comment by orta** [comment]
Is prerelease one word? I'd have thought it was pre-release?

**Comment by floere** [comment]
@orta RubyGems uses `prerelease`, and semver uses `pre-release`. I've decided to go with RubyGems, but I don't feel strongly either way, so I am very open to having that changed.

**Comment by floere** [comment]
@kylef What would be on the before/after screenshot – results with and without prerelease versions? Maybe it's ok that it's just a small post?

**Comment by AliSoftware** [comment]
I think we should conclude with the following example :santa: 
```
# Before the change
$ pod install holidays
Installing Holidays (2.0.15)
# After the change
$ pod install holidays
Installing Holidays (2.0.16-pre)
```

File: _posts/2015-12-03-Release-Versions-Only.markdown [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+This sounds good in principle, but the behaviour is at odds with how the `pod` command operates.
+The default there is to use the latest _released_ pod version.
+As a result of this – as we noticed via our stats system – usage of prerelease versions is minimal, and showing the latest prerelease version on [cocoapods.org](https://cocoapods.org) violates expectations as to what `pod install` will actually install.
+
+Therefore, we will be switching to the `pod` command default also on [cocoapods.org](https://cocoapods.org), meaning in the results, we will show only released versions.
+We've prepared all systems for the switch and will flip it on December 27th 2015 during the holiday lull.
+
+Happy Holidays, pod friends!
\ No newline at end of file
```
Comment (segiddins) [inline _posts/2015-12-03-Release-Versions-Only.markdown]: Trailing newline

**Comment by segiddins** [comment]
👍🏻

**Comment by orta** [comment]
:+1:

**Comment by floere** [comment]
@AliSoftware Nothing changes though in the pod command :)

**Comment by kylef** [comment]
@floere I was just thinking of something like:

Before:

screen shot 2015-12-22 at 09 36 22

After:

screen shot 2015-12-22 at 09 37 26

**Comment by AliSoftware** [comment]
@floere ah indeed then maybe a capture of the website before/after with the opposite joke then ^^

**Comment by floere** [comment]
@kylef @AliSoftware I like your ideas for adding images/jokes – in this case I'd like to keep it snappy and above the fold, though.

**Comment by floere** [comment]
http://blog.cocoapods.org/Release-Versions-Only/

**Comment by kylef** [comment]
> "above the fold"

Didn't quite work :wink: 

2016-01-02-193821_1366x768_scrot

**Comment by floere** [comment]
@kylef "Above the 'Read On' fold on the main page" is what I meant but failed to write. Such extravagant use of whitespace! :)

**Comment by orta** [comment]
this is the _only_ way
bwahhahaa

**Comment by floere** [comment]
Ok

**Pull request merged**